lib/components/slider/index.tsx

```diff
@@ -6,24 +6,34 @@ type Props = {
   min: number;
   max: number;
   value: number;
+  list: string;
 };
 
 export const Slider: FunctionComponent<Props> = ({
   disabled,
   min,
   max,
   value,
+  list,
   onChange,
 }) => (
-  <input
-    className="slider"
-    disabled={disabled}
-    type="range"
-    min={min}
-    max={max}
-    value={value}
-    onChange={onChange}
-  />
+  <>
+    <input
+      className="slider"
+      disabled={disabled}
+      type="range"
+      min={min}
+      max={max}
+      value={value}
+      list={list}
+      onChange={onChange}
+    />
+    <datalist id={list}>
+      {_.range(min, max + 1).map((value, index) => {
```

Comment:
let's avoid assuming that a global `_` exists. I didn't realize this is the case - does this work or crash?

if you want `lodash.range` it can be imported at the top of the module

```ts
import {range} from 'lodash';
```

Reply:
Thanks @dmsnell. This did work I used the autocomplete, but I do see a warning in VS Code about importing as well. I'm going to end up removing this from this PR before it's ready to go. In the future I'll be sure to explicitly import.